### PR TITLE
Updated release notes for ACPCampaignClassic v2.1.1

### PR DESCRIPTION
**Pull request by swarna04**

### CODE REVIEW

File: release-notes/README.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## May 17, 2022
+
+### iOS Campaign Classic 2.1.1
+
+* ACPCampaignClassic iOS SDK is now ACC v8 compatible! BroadlogId can be provided in the UUID format in the notification tracking APIs.
```
Comment (calebk1) [inline release-notes/README.md:11]: Just to confirm - what is ACC?

Adobe Campaign Classic?

**Review by calebk1 — COMMENTED** [review]
Just one question.

File: release-notes/README.md (hunk 1 above)
Comment (swarna04) [reply release-notes/README.md:11]: ah yes! Should I expand it?

Comment (swarna04) [reply release-notes/README.md]: Updated the pr @calebk1 , let me know if it looks fine now.

**Review by calebk1 — APPROVED** [review]
LGTM!

**Pull request merged**